### ISSUE TITLE
Xcode 12 compiling issue. openURL is deprecated.

### ISSUE THREAD

**Issue by par400** [square/SquarePointOfSaleSDK-iOS#86]
I installed the SDK  via cocoapods and run on xcode 12 beta 4 but the compiler complains about this function, `_performRequestWithURL` return statement of the `SCCAPIConnection.m` file

```
+ (BOOL)_performRequestWithURL:(nonnull NSURL *)URL error:(out NSError *__nullable *__nullable)error;
{
    if (![self _canPerformRequestWithURL:URL error:error application:[UIApplication sharedApplication]]) {
        return NO;
    }

    return [[UIApplication sharedApplication] openURL:URL]; // -> openURL is deprecated in ios 10.0...
}
```
when I do the conversion to the suggested change, it still gives error after.
this is the error before changing:
<img width="1023" alt="openurl1" src="https://user-images.githubusercontent.com/66535353/94348592-6501b280-fff2-11ea-9551-81f01fd62d4a.png">

this is after changing but still gives a weird objective-c error:
<img width="1032" alt="openurl2" src="https://user-images.githubusercontent.com/66535353/94348606-806cbd80-fff2-11ea-9067-736e981965e1.png">

any thoughts on this would be appreciated

**Comment by MikeSilvis** [comment]
https://github.com/square/SquarePointOfSaleSDK-iOS/pull/87

**Comment by MikeSilvis** [comment]
@par400 this should now be resolved with 3.4.1
